### ISSUE TITLE
Have an alias to use dryrun=True instead of test=True

### ISSUE THREAD

**Issue by arthurlogilab** [saltstack/salt#45376]
### Description of Issue/Question

Some new salt users I came across told me that the test=True could be confusing and would prefer to have a more standard dryrun terminology for what test=True does in salt. I thought I'd start the discussion here about having an alias that does this. 

Is it technically difficult to have an alias for dryrun=test which adds test=True early enough to provide such a functionality ? 

Doing a grep for dryrun in the code I find that some states already use this argument so if we go down that road we need to make sure they don't break (pkgng, rsync, dockermod)

**Comment by garethgreenaway** [comment]
@thatch45 Would you weighing on this one?

**Comment by thatch45** [comment]
`dryrun` or `dry_run` or `no-op` etc would all be good options here. All we would need to do to add an alias is just take the argument to state.apply/highstate/sls/high/low/pkg functions, would be to add this:

```
def apply(mods=None, **kwargs):
    if 'dryrun' in kwargs:
        kwargs['test'] = kwargs.pop('dryrun')
```
then it would be 100% safe

**Comment by garethgreenaway** [comment]
@thatch45 Thanks!  Adding the Feature label.

**Comment by thatch45** [comment]
Yes, this is something that would take all of 15 minutes to add and we should have added it long ago

**Comment by arthurlogilab** [comment]
up

**Comment by arthurlogilab** [comment]
up
